src/ble/CHIPBleServiceData.h

```diff
@@ -45,11 +45,12 @@ enum chipBLEServiceDataType
  */
 struct ChipBLEDeviceIdentificationInfo
 {
-    constexpr static uint16_t kDiscriminatorMask     = 0xfff;
-    constexpr static uint8_t kAdditionalDataFlagMask = 0x1;
+    constexpr static uint16_t kDiscriminatorMask        = 0xfff;
+    constexpr static uint8_t kAdditionalDataFlagMask    = 0x1;
+    constexpr static uint16_t kAdvertisementVersionMask = 0xf000;
 
     uint8_t OpCode;
-    uint8_t DeviceDiscriminator[2];
+    uint8_t DeviceDiscriminator[2]; // 12 bits for device discriminator and 4 bits for advertisement version
```

Comment:
Should it be renamed accordingly too? So that it is understood that it has device discriminator as well as advertisement version?

Reply:
It may be used by other code, so didn’t changed.

Reply:
Is the other code directly using this variable? It should do it through the Get/Set APIs, right?

Reply:
Yes